meta-refkit-extra/recipes-convnet/boost/boost_%.bbappend

```diff
@@ -0,0 +1 @@
+PACKAGECONFIG += "python"
```

Comment:
That's still needed -- it enables Python support in boost. We no longer need to select Python 3 by editing the `user-config.jam` file, because Python 3 support is now enabled as default if it's found. See difference to the previous iteration of the same file. https://github.com/ipuustin/intel-iot-refkit/blob/165b46149a2c5a58788b91d2cac97c394e567f14/meta-refkit/recipes-convnet/boost/boost_%25.bbappend

However, it's a good question if we should move the `PACKAGECONFIG` variable to the layer distro configuration file instead of the `.bbappend` file. What do you think?